controllers/storagecluster/cephobjectstores_test.go

```diff
@@ -47,6 +47,13 @@ func TestCephObjectStores(t *testing.T) {
 
 func assertCephObjectStores(t *testing.T, reconciler StorageClusterReconciler, cr *api.StorageCluster, request reconcile.Request) {
 	expectedCos, err := reconciler.newCephObjectStoreInstances(cr, nil)
+	expectedCos[0].Spec.MetadataPool.Parameters = map[string]string{
+		"bulk": "true",
+	}
+	expectedCos[0].Spec.DataPool.Parameters = map[string]string{
+		"bulk": "true",
+	}
+
 	assert.NoError(t, err)
```

Comment:
Is there a test that checks that `bulk` isn't applied to already-existing stores? That is also important behavior.

Reply:
Will add soon

Reply:
Added detailed unit tests